bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/AbstractTextEditor.java

```diff
@@ -3366,6 +3373,8 @@ public void createPartControl(Composite parent) {
 		int styles= SWT.V_SCROLL | SWT.H_SCROLL | SWT.MULTI | SWT.BORDER | SWT.FULL_SELECTION;
 		fSourceViewer= createSourceViewer(parent, fVerticalRuler, styles);
 
+		fSourceViewer.getTextWidget().setData(DISABLE_CSS, Boolean.TRUE);
```

Comment:
In @vogella 's propose there was a nice comment why it's ok to disable the styling. Can you add a similar comment?

Reply:
Done